src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -849,6 +851,10 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		if (!this.xterm) {
 			return;
 		}
+		const runRecentStorageKey = `${TerminalStorageKeys.PinnedRecentCommandsPrefix}.${this._shellType}`;
+		if (!runRecentStorageKey) {
```

Comment:
This if could never be false because of the `.` in the `runRecentStorageKey` variable. What is this check needed for?

Reply:
good catch - that was leftover